### PR TITLE
Feature/feature tr change password

### PR DESCRIPTION
**Pull request by enrichp**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrDeleteUser.java  3
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrChangePassword.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5239609&bid=17173497)

File: app/src/main/java/org/pesmypetcare/mypetcare/controllers/ControllersFactory.java [hunk 1]
```diff
@@ -12,4 +12,25 @@ public static TrRegisterNewPet createTrRegisterNewPet() {
     public static TrUpdatePetImage createTrUpdatePetImage() {
```
Comment (AlbertPG) [inline app/src/main/java/org/pesmypetcare/mypetcare/controllers/ControllersFactory.java:12]: This class will be commented whenever all controllers of the iteration are included

**Review by AlbertPG — APPROVED** [review]
No problem at all

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrChangePassword.java  3
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrDeleteUser.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5239609&bid=17173497)

**Review by xavier-campos-diaz — APPROVED** [review]

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrChangePassword.java  3
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrDeleteUser.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5239609&bid=17173497)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrDeleteUser.java  3
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrChangePassword.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5239609&bid=17173497)

**Pull request merged**